tensorflow/lite/delegates/gpu/gl/gl_errors.cc

```diff
@@ -67,6 +67,10 @@ absl::Status GetOpenGlErrors() {
   if (error2 == GL_NO_ERROR) {
     return absl::InternalError(ErrorToString(error));
   }
+  EGLContext context = eglGetCurrentContext();
```

Comment:
Is this the right location for checking?  shouldn't this check be at L.62, before the first glGetError?

Reply:
https://www.khronos.org/opengl/wiki/OpenGL_Error
Note: No OpenGL function call is valid when an OpenGL Context is not made current. Depending on the platform, a call to glGetError without having a context made current may crash or return any value, including indefinitely returning a valid OpenGL error code. So if you see an infinite loop in such a function, this is likely the reason why.

Hmmm, I move it upper.